tools/test-setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -xe
```

Comment:
This isn't needed any more we've landed changes to zuul, to always use the latest version of podman.

Reply:
Tracked with issue https://github.com/ansible/ansible-navigator/issues/126